loki-build-image/Dockerfile

```diff
@@ -97,8 +97,8 @@ RUN dpkg --add-architecture armhf && \
 
 COPY --from=docker /usr/bin/docker /usr/bin/docker
 COPY --from=docker /usr/libexec/docker/cli-plugins/docker-buildx /usr/libexec/docker/cli-plugins/docker-buildx
-COPY --from=helm /usr/bin/helm /usr/bin/helm
-COPY --from=helm /go/bin/helm-docs /usr/bin/helm-docs
+COPY --from=helm /usr/local/bin/helm /usr/bin/helm
+COPY --from=helm /usr/local/bin/helm-docs /usr/bin/helm-docs
```

Comment:
why we need this change? curious how was it working before?

Reply:
See line 13 and 16. I had to download the proper release. I guess for helm-docs it could have still worked with go modules.